src/main/java/spoon/reflect/factory/QueryFactory.java

```diff
@@ -48,4 +48,12 @@ public CtQuery createQuery() {
 	public CtQuery createQuery(Object input) {
 		return new CtQueryImpl(input);
 	}
+
+	/**
+	 * Creates a bound query with an optional number of inputs
```

Comment:
what is the meaning of inputs?

not fan of pointing to an implementation in an interface.